SlicerCART/src/SlicerCART.py

```diff
@@ -19,17 +18,58 @@
 from bids_validator import BIDSValidator
 import filecmp
 import shutil
-import nib
+import nibabel as nib
```

Comment:
that will probably fail if the library is not installed yet. You probably want to move these imports after the call to `check_and_install_python_packages()`